static/app/components/feedback/feedbackItem/activitySection.tsx

```diff
@@ -0,0 +1,300 @@
+import {Fragment, useState} from 'react';
+
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  clearIndicators,
+} from 'sentry/actionCreators/indicator';
+import {Client} from 'sentry/api';
+import {ActivityAuthor} from 'sentry/components/activity/author';
+import {ActivityItem} from 'sentry/components/activity/item';
+import {Note} from 'sentry/components/activity/note';
+import {NoteInputWithStorage} from 'sentry/components/activity/note/inputWithStorage';
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import useMutateFeedback from 'sentry/components/feedback/useMutateFeedback';
+import * as Layout from 'sentry/components/layouts/thirds';
+import ReprocessedBox from 'sentry/components/reprocessedBox';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import {
+  Group,
+  GroupActivityNote,
+  GroupActivityReprocess,
+  GroupActivityType,
+  Note as NoteType,
+  Organization,
+  User,
+} from 'sentry/types';
+import {uniqueId} from 'sentry/utils/guid';
+import withApi from 'sentry/utils/withApi';
+import withOrganization from 'sentry/utils/withOrganization';
+import GroupActivityItem from 'sentry/views/issueDetails/groupActivityItem';
+import {
+  getGroupMostRecentActivity,
+  getGroupReprocessingStatus,
+  ReprocessingStatus,
+} from 'sentry/views/issueDetails/utils';
+
+// This file is based on issueDetails/groupActivity.tsx
+
+type Props = {
+  api: Client;
+  group: Group;
+  organization: Organization;
+};
+
+function deleteNote({
+  group,
+  id,
+  organization,
+  api,
+  deleteComment,
+}: {
+  api: Client;
+  deleteComment;
+  group: Group;
+  id: string;
+  organization: Organization;
+}) {
+  const promise = api.requestPromise(
+    `/organizations/${organization.slug}/issues/${group.id}/comments/${id}/`,
+    {
+      method: 'DELETE',
+    }
+  );
+
+  const newActivity = group.activity.filter(a => a.id !== id);
+  deleteComment(newActivity);
```

Comment:
I think there's flashing on the screen because we have both `api.requestPromise()` as well as `deleteComment()`

deleteComment() seems to be the update mutate callback from `useMustateFeedback`.